ftplugin/org.lua

```diff
@@ -10,6 +10,12 @@ config:setup_mappings('org')
 config:setup_mappings('text_objects')
 config:setup_foldlevel()
 
+if config.org_startup_indented then
+  config.org_adapt_indentation = not config.org_indent_mode_turns_off_org_adapt_indentation
```

Comment:
Let's move this line to `Config:extend` where we also do some validation and changes to the config.

Reply:
Will do. I do have to keep the conditional there though, as we use the `ftplugin` to set the buffer `org_indent_mode` variable as a heads up.